ui/src/overview.tsx

```diff
@@ -118,8 +126,47 @@ function AddCriteriaButton(props: { group: string | GroupKind }) {
 }
 
 function ParticipantsGroup(props: { group: Group }) {
+  const dispatch = useAppDispatch();
+  const cohort = useCohort();
+  const [renameGroupDialog, showRenameGroup] = useTextInputDialog({
```

Comment:
@tjennison-google  I found a bug in TNG-17 rename dialog. there are 3 parameters in the useTextInputDialog. But we need another parameter for the text. or it will cause the dialog default show "title" at the first time when we modify the name. Like this: 
![Screen Shot 2022-04-21 at 10 43 51 PM](https://user-images.githubusercontent.com/100877272/164585912-a076357e-1c8c-44e4-9c10-e7ce597f09bf.png)

But this is only for the first time, next time, it will show the last changed name.
![Screen Shot 2022-04-21 at 10 48 40 PM](https://user-images.githubusercontent.com/100877272/164586441-8e05f734-3520-4bc0-806c-a94e3ca401fe.png)

 I checked All of Us page. It shows empty in the textInputDialog at the first time to modify the name.  
<img width="1011" alt="Screen Shot 2022-04-21 at 10 45 13 PM" src="https://user-images.githubusercontent.com/100877272/164586065-79ec3f8c-8dbb-4ecf-ba7a-ce1e9c2db9fe.png">
 
Should I use the default empty blank that is consistent with All of us? 
--yes, I will create a patch PR for it
--no, should we show the actual name in the blank? like shows "Group1" while it modify at the first time. 

I also notice there is a red warning if I enter only spaces. I think I also need to add this corner case in the patch PR.
<img width="500" alt="Screen Shot 2022-04-21 at 11 02 18 PM" src="https://user-images.githubusercontent.com/100877272/164587961-5da9c9f7-3881-4ef7-a503-2a97b6a05712.png">

Reply:
Don't worry about the input box starting blank. AoU has the current name in the title instead but, in my opinion at least, it's better to have the current name editable.

Also don't worry about the red error. The confirm button is disabled when the input box is empty which is the most important part. It certainly wouldn't hurt to add it but that can easily go into a separate PR, though we would use MUI's input error UI which changes the color of the label and border or something like that.